qiskit_machine_learning/kernels/trainable_kernel_mixin.py

```diff
@@ -0,0 +1,78 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Trainable Quantum Kernel"""
```

Comment:
Maybe this can be altered to note its a mixin here - otherwise its the exact same as the TrainableFidelityQKernel

Reply:
Updated